packages/legend-application-studio/src/stores/EditorStore.ts

```diff
@@ -169,8 +135,8 @@ export class EditorStore implements CommandRegistrar {
   sdlcState: EditorSDLCState;
   graphState: EditorGraphState;
   graphManagerState: GraphManagerState;
+  graphEditModeManagerState: GraphEditorModeManagerState;
```

Comment:
im thinking of a more succinct name for this. Since calling `editorStore.graphEditModeManagerState.graphEditModeState` everywhere seems quite laborous, maybe what we should do is to squash `GraphEditorModeManagerState` into `EditorStore` and call `graphEditModeState` `graphEditorMode`, so we simply calling `editorStore.graphEditorMode`

Reply:
eventually, we can rename/remove `EditorMode` and this should be the defacto `EditorMode` - https://github.com/finos/legend-studio/issues/317